mmv1/products/compute/Subnetwork.yaml

```diff
@@ -388,3 +395,17 @@ properties:
     output: true
     description: |
       The range of external IPv6 addresses that are owned by this subnetwork.
+  - !ruby/object:Api::Type::Boolean
+    name: 'allowSubnetCidrRoutesOverlap'
+    default_from_api: true
+    update_verb: :PATCH
+    update_url: projects/{{project}}/regions/{{region}}/subnetworks/{{name}}
+    fingerprint_name: 'fingerprint'
+    update_id: 'allowSubnetCidrRoutesOverlap'
```

Comment:
Can this variable no be updated with the other parameters?

We can omit these from the field level specification if so

Reply:
Ah, got it. Let me check.

Reply:
I confirmed this can be updated with a PATCH with other parameters (as long as the fingerprint is provided). But I found out that disabling it was not working since I was skipping sending a false value on updates. Should be good now.